jumanji/environments/combinatorial/cvrp/specs_test.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+import chex
+import jax.numpy as jnp
+import pytest
+
+from jumanji.environments.combinatorial.cvrp.env import CVRP
+from jumanji.environments.combinatorial.cvrp.types import Observation
+
+
+class TestObservationSpec:
+    env = CVRP()
+    observation_spec = env.observation_spec()
+    observation = observation_spec.generate_value()
```

Comment:
We should avoid defining variables at the class level as we add in the possibility that they get changed during testing. Instead we should put them in a setup() that is called for each test or we put them in as fixtures. Happy to go with either. My preference is fixtures